app/assets/stylesheets/components/_native-campaign.scss

```diff
@@ -0,0 +1,32 @@
+$dark-grey: #999;
+
+.app-c-native-campaign__container {
+  border-top: 1px solid $border-colour;
+  padding-top: 20px;
+}
+
+.app-c-native-campaign__header {
+  @include core-14;
+  color: $dark-grey;
+  display: inline-block;
+  padding: 4px 6px 1px;
+  border: 1px solid $border-colour;
+  border-radius: 4px;
+}
+
+.app-c-native-campaign__link {
+  margin-top: 15px;
+  display: block;
+  text-decoration: none;
+  @include core-19;
+  font-weight: bold;
```

Comment:
If you want it to be bold and 19px, you can include `bold-19` instead of `core-19`. This means you won't need `font-weight: bold`